nep-NoVoteNoSysFees.mediawiki

```diff
@@ -0,0 +1,39 @@
+<pre>
+  NEP: <to be assigned>
+  Title: No vote, No System Fees
+  Author: Malcolm Lerider
+  Type: Standard
+  Status: Draft
+  Created: 2017-11-02
+</pre>
+
+==Abstract==
+
+This NEP suggests an encouragement for NEO holders to vote for consensus nodes. 
+
+GAS distribution is divided into two parts: 1. Generation of new GAS by NEO tokens and 2. Recycled system fees. In the current implementation, NEO holders will automatically receive both these distributions as unclaimed GAS. This NEP suggests that only generation of new GAS should be automatically distributed to NEO. To be eligible to receive recycled system fees, the address holding the NEO must have voted for consensus nodes. How the address has voted does not affect how much of the recycled system fees they get; it is distributed proportionally to all NEO that are held in an address that has voted.
```

Comment:
@lerider, @vang1ong7ang mentioned that we may still faill into the prisionner dilema if we keep distributing the "inflation".

> This NEP suggests that only generation of new GAS should be automatically distributed to NEO

In this sense, if the recycled system fees are dropped to a low level, voter will have no additional incentives (willing to vote).